bcipy/display/rsvp/display.py

```diff
@@ -364,32 +512,64 @@ def wait_screen(self, message, color):
         wait_message.draw()
         self.window.flip()
 
-    def create_stimulus(self, height_int: int, mode: str = 'text'):
+    def _create_stimulus(
```

Comment:
Rather than a large if-else clause, I would prefer to see separate methods for each mode. However, it would take some thought on how to avoid repeating the default parameters.